webview_panels/src/lib/altimate/main.d.ts

```
@@ -0,0 +1,597 @@
+    frontend_url: string;
+}
+
+export declare interface CoachAiResponse {
+    ai_response: string;
+    category: string;
+    personalizationScope: string;
+}
+
+export declare const CoachForm: ({ taskLabel, context, onClose }: Props_10) => JSX_2.Element;
+
+export declare const CoachFormButton: ({}: Props_11) => JSX_2.Element;
```

Biome

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)